scripts/asm-installer/install_asm

```diff
@@ -1715,6 +1755,20 @@ install_secrets() {
     --from-file="${CA_CHAIN}"
 }
 
+init_private_ca() {
+  WORKLOAD_IDENTITY="$PROJECT_ID.svc.id.goog[istio-system/istiod-service-account]"
```

Comment:
I don't think this needs to be a variable. I'd prefer it be a function:

```
workload_identity() {
  echo "${PROJECT_ID}.svc.id.goog[istio-system/istiod-service-account]"
}
```

If it needs to be a variable for some reason, please use local.

Reply:
Prefer it remains a local function. May be built from more local variables.

Reply:
fixed